src/tests/ftest/launch.py

```diff
@@ -911,8 +911,39 @@ def archive_logs(avocado_logs_dir, test_yaml, args):
 
     # Copy any log files written to the DAOS_TEST_LOG_DIR directory
     logs_dir = os.environ.get("DAOS_TEST_LOG_DIR", DEFAULT_DAOS_TEST_LOG_DIR)
+
     archive_files(destination, host_list, "{}/*.log*".format(logs_dir))
 
+    # Caution: the glob expression "_output.log" must match the
+    #   --output-filename specified in # cart_utils.py:get_env()
+    tar_files(destination, "{}/*_output.log*".format(logs_dir))
+
+def tar_files(destination, logs_glob_expr):
+    """Run tar on files (in the directory specified by the orterun
+        --output-filename option) to the avocado results directory.  Useful for
+        copying, e.g., files from orterun --output-filename option to the avocado
+        results directory.
+
+    Args:
+        destination (str): (glob expression) path of files to be tar'd
+        logs_dir (str): destination of tar file(s)
```

Comment:
These arguments don't seem to match the actual arguments in the function definition.